operator/src/test/java/org/keycloak/operator/testsuite/unit/PodTemplateTest.java

```diff
@@ -642,4 +660,36 @@ public void testAffinity() {
         assertThat(podTemplate.getSpec().getAffinity()).isNotEqualTo(affinity);
     }
 
+    private Job getUpdateJob(Consumer<KeycloakSpecBuilder> newSpec, Consumer<KeycloakSpecBuilder> oldSpec, Consumer<StatefulSetBuilder> exsitingModifier) {
```

Comment:
Just a nitpick, but we're fine to merge as is.
```suggestion
    private Job getUpdateJob(Consumer<KeycloakSpecBuilder> newSpec, Consumer<KeycloakSpecBuilder> oldSpec, Consumer<StatefulSetBuilder> existingModifier) {
```